packaging/ubuntu-16.04/snapd.postinst

```diff
@@ -39,16 +39,6 @@ case "$1" in
             done
         fi
 
-        # In commit 0dce4704a5d (2017-03-28, snapd v2.23.6) we renamed
-        # /etc/apparmor.d/usr.lib.snap-confine to usr.lib.snap-confine.real
-        # to fix LP: #1673247 - however some people (upgrades?) still have
-        # the old usr.lib.snap-confine file. This seems to be loaded instead
-        # of the correct usr.lib.snap-confine.real profile. To fix this we
-        # use the rather blunt approach to remove the file forcefully.
-        if test -f /etc/apparmor.d/usr.lib.snapd.snap-confine && test -f /etc/apparmor.d/usr.lib.snapd.snap-confine.real; then
-            rm -f /etc/apparmor.d/usr.lib.snapd.snap-confine
-        fi
```

Comment:
I'm missing an understanding of why this rm -f wasn't already sufficient to solve the reported bugs.  Using rm_conffile may be "cleaner", but how does switching the method help the user-facing runtime behavior?

Reply:
Aha, thank you! If its considered okay to use this method I will change this PR to use rm (or mv to .dpkg-bak) and leave the rm_conffile calls alone. Will push that shortly.